src/@lekoarts/gatsby-theme-minimal-blog/components/page.tsx

```diff
@@ -1,26 +1,41 @@
 /** @jsx jsx */
-import type { HeadFC, PageProps } from "gatsby"
-import { jsx, Heading } from "theme-ui"
-import Layout from "./layout"
-import Seo from "./seo"
+import type { HeadFC, PageProps } from 'gatsby';
+import { jsx, Heading } from 'theme-ui';
+import Layout from './layout';
+import Seo from './seo';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 export type MBPageProps = {
   page: {
-    title: string
-    slug: string
-    excerpt: string
-  }
-}
+    title: string;
+    slug: string;
+    excerpt: string;
+    ns: string;
+  };
+};
 
-const Page: React.FC<React.PropsWithChildren<PageProps<MBPageProps>>> = ({ data: { page }, children }) => (
-  <Layout>
-    <Heading as="h2" variant="styles.h2">
-      {page.title}
-    </Heading>
-    <section sx={{ my: 5, variant: `layout.content` }}>{children}</section>
-  </Layout>
-)
+type PageContext = {
+  frontmatter: { ns: string };
+};
 
-export default Page
+const Page: React.FC<
+  React.PropsWithChildren<PageProps<MBPageProps, PageContext>>
+> = ({ children, data }) => {
+  const ns = data?.page?.ns;
+  const { t } = useTranslation(ns);
+  return (
+    <Layout>
+      <Heading as="h2" variant="styles.h2">
+        {t('title')}
+      </Heading>
+      <section sx={{ my: 5, variant: `layout.content` }}>{children}</section>
+    </Layout>
+  );
+};
 
-export const Head: HeadFC<MBPageProps> = ({ data: { page } }) => <Seo title={page.title} description={page.excerpt} />
\ No newline at end of file
+export default Page;
+
+export const Head: HeadFC<MBPageProps> = ({ data: { page } }) => (
+  <Seo title={page.title} description={page.excerpt} />
```

Comment:
@kaleemullahdev Page metedata needs to be translated as well;
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/b448d1ec-614a-4687-bd85-1c4aa086f2b6)